src/applications/mhv/secure-messaging/tests/e2e/secure-messaging-compose-reply-with-attachment.cypress.spec.js

```diff
@@ -0,0 +1,89 @@
+import PatientMessagesLandingPage from './pages/PatientMessagesLandingPage';
+import PatientComposePage from './pages/PatientComposePage';
+import manifest from '../../manifest.json';
+
+describe(manifest.appName, () => {
+  it('can send message', () => {
+    const landingPage = new PatientMessagesLandingPage();
+    const composePage = new PatientComposePage();
+    landingPage.login();
+    landingPage.loadPage(false);
+    cy.get('[data-testid="compose-message-link"]').click();
+    cy.injectAxe();
+    cy.axeCheck();
+    cy.get('[data-testid="compose-recipient-select"]')
+      .shadow()
+      .find('[id="select"]')
+      .select('BLUE ANCILLARY_TEAM');
+    cy.get('[name="COVID"]').click();
+    cy.get('[data-testid="attach-file-input"]').selectFile(
+      'src/applications/mhv/secure-messaging/tests/e2e/fixtures/test_video.mp4',
+      { force: true },
+    );
+
+    cy.get('[data-testid="attach-file-error-modal"] p')
+      .should(
+        'have.text',
+        'File supported: doc, docx, gif, jpg, jpeg, pdf, png, rtf, txt, xls, xlsx',
+      )
+      .should('be.visible');
+    cy.get('[data-testid="attach-file-error-modal"]')
+      .shadow()
+      .find('[type="button"]')
+      .first()
+      .click();
+    cy.get('[data-testid="attach-file-input"]').selectFile(
```

Comment:
Can you encapsulate the code for attaching a message into a method :  PatientComposeMessage.attachMessageFromFile(<<filePath>>)

Reply:
yes, encapsulated the code for attaching a message into a method.